web/src/people.js

```diff
@@ -654,7 +654,7 @@ export function exclude_me_from_string(user_ids_string) {
 }
 
 export function format_small_avatar_url(raw_url) {
-    const url = new URL(raw_url, location);
+    const url = new URL(raw_url, location.origin);
```

Comment:
This code definitely works, and the stringification is documented: https://developer.mozilla.org/en-US/docs/Web/API/URL/URL

I think it's fine to do this change but the commit message looks to be wrong.